doc/en/components/scheduling/date-range-picker.md

```diff
@@ -0,0 +1,418 @@
+For example, to display the date in a Japanese format, you can set the locale property like this:
+```html
+<igc-date-range-picker locale="ja-JP">
+</igc-date-range-picker>
+```
+
+If you want to manually define the date format, you can use the `InputFormat` property by passing a custom format string:
+
+```html
+<igc-date-range-picker input-format="dd/MM/yy">
+</igc-date-range-picker>
+```
+
+The `DisplayFormat` property also accepts a custom format string, but it only applies when the input field is idle (i.e., not focused). When the field is focused, the format reverts to the default or to the one defined by `InputFormat`, if both properties are used together:
+
+```html
+<igc-date-range-picker input-format="dd/MM/yy" date-format="yy/MM/dd">
+</igc-date-range-picker>
+```
+
+### Calendar Layout and Formatting
+
+You can further customize the pop-up calendar using various properties:
+
+* The `Orientation` property allows you to set whether the calendar should be displayed vertically or horizontally.
+
+* The `VisibleMonths` property controls how many months are visible at a time, with a value of either 1 or 2.
+
+* The `ShowWeekNumbers` property enables or disables the week number column in the calendar.
+
+```html
+<igc-date-range-picker orientation="vertical" visible-months="1" show-week-numbers="true">
+</igc-date-range-picker>
+```
+
+### Min & Max
+
+You can also set the `Min` and `Max` properties to restrict user input by disabling calendar dates outside the defined range. These properties act as validators, so even if the user manually types a date outside the range, the `DateRangePicker` will become invalid.
+
+```html
+<igc-date-range-picker min="2025-05-06" max="2025-05-10">
+</igc-date-range-picker>
+```
+
+### Custom Date range
+
+You can also add custom date range chips to the calendar pop-up for faster range selection using the `CustomRanges` property. For example, you can create a custom date range chip to quickly select the range for the previous 7 days, ending with the current date.
+
+```ts
+const today: Date = new Date();
+
+const previousSeven = new Date(
+  today.getFullYear(),
+  today.getMonth(),
+  today.getDate() - 7 
+);
+const lastWeek: CustomDateRange[] = [
+  {
+    label: 'Previous 7 days',
+    dateRange: {
+      start: previousSeven,
+      end: today,
+    },
+  }
+]
+
+const dateRange = document.querySelector('igc-date-range-picker') as IgcDateRangePickerComponent;
+dateRange.customRanges = lastWeek
+```
+
+Now, when you click the newly created `"Previous 7 days"` chip in the calendar pop-up, the range will automatically be selected, starting from 7 days ago and ending today.
+
+`sample="/scheduling/date-range-picker/custom-ranges", height="500", alt="{Platform} Date Range Picker Custom Ranges Example"`
+
+### Forms
+
+The `DateRangePicker` component can also be used seamlessly with the HTML form element. The `Min`, `Max`, and `Required` properties act as form validators.
+
```

Comment:
Maybe add info about configuring the disabled dates, i.e.:
```ts
const disabledDates: DateRangeDescriptor[] = [
  {
    type: DateRangeType.Between,
    dateRange: [minDate, maxDate],
  },
];
```
or link this calendar topic [section](https://www.infragistics.com/products/ignite-ui-web-components/web-components/components/scheduling/calendar#disabled-dates), where more details about the `DateRangeDescriptor` objects are given.